api/src/main/java/io/opentelemetry/correlationcontext/CorrelationsContextUtils.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.correlationcontext;
+
+import io.grpc.Context;
+import io.opentelemetry.context.ContextUtils;
+import io.opentelemetry.context.Scope;
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * Utility methods for accessing the {@link CorrelationContext} contained in the {@link
+ * io.grpc.Context}.
+ *
+ * <p>Most code should interact with the current context via the public APIs in {@link
+ * CorrelationContext} and avoid accessing this class directly.
+ *
+ * @since 0.1.0
+ */
+@Immutable
+public final class CorrelationsContextUtils {
+  private static final Context.Key<CorrelationContext> CORR_CONTEXT_KEY =
+      Context.key("opentelemetry-corr-context-key");
+
+  /**
+   * Creates a new {@code Context} with the given value set.
+   *
+   * @param corrContext the value to be set.
+   * @return a new context with the given value set.
+   * @since 0.1.0
+   */
+  public static Context withCorrelationContext(CorrelationContext corrContext) {
+    return withCorrelationContext(corrContext, Context.current());
+  }
```

Comment:
Let's remove this for the moment, people can use the other version.

Reply:
Keep the version returning a `Scope`?